### PR TITLE
SAK-48533 - Add explicit depends-on to bean(s) that depend on localization properties

### PR DESCRIPTION
**Pull request by jonespm**
I'm only modifying the one in content as the tests fail adding it to kernel.

Perhaps there's a better way to do this but this at least verified that things were working and the load order is correct there. This also could just be closed if @ern doesn't think it's necessary. 

Currently draft until I get some more time to test/think about it.

### CODE REVIEW

**Review by ottenhoff — APPROVED** [review]
Makes sense to me. @ern?

**Comment by ern** [comment]
I am looking at what's going on with this

**Comment by jonespm** [comment]
This isn't required, but if this was here it wouldn have been more obvious that this was broken when this component deployment was changed. 

All of these files depend on loading this class locally and some of these are used by this service. Actually it looks like it's just TypeProperties and not ContentTypeProperties. 

```
content-types/src/java/org/sakaiproject/content/types/FolderType.java
81:	private static final String DEFAULT_RESOURCECLASS = "org.sakaiproject.localization.util.TypeProperties";

content-types/src/java/org/sakaiproject/content/types/HtmlDocumentType.java
56:	private static final String DEFAULT_RESOURCECLASS = "org.sakaiproject.localization.util.TypeProperties";

content-types/src/java/org/sakaiproject/content/types/FileUploadType.java
71:	private static final String DEFAULT_RESOURCECLASS = "org.sakaiproject.localization.util.TypeProperties";

content-types/src/java/org/sakaiproject/content/types/TextDocumentType.java
60:	private static final String DEFAULT_RESOURCECLASS = "org.sakaiproject.localization.util.TypeProperties";

content-types/src/java/org/sakaiproject/content/types/UrlResourceType.java
55:	private static final String DEFAULT_RESOURCECLASS = "org.sakaiproject.localization.util.TypeProperties";

content-types/src/java/org/sakaiproject/content/types/MakeSitePageAction.java
46:	private static final String DEFAULT_RESOURCECLASS = "org.sakaiproject.localization.util.TypeProperties";
```

File: content/content-types/src/webapp/WEB-INF/components.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
 
 	<bean id="ContentTypeRegistyBean" class="org.sakaiproject.content.types.ContentTypeRegistryBean"
-	init-method="init">
+	init-method="init" depends-on="org.sakaiproject.localization.util.ContentTypeProperties">
```
Comment (jonespm) [inline content/content-types/src/webapp/WEB-INF/components.xml:7]:
```suggestion
	init-method="init" depends-on="org.sakaiproject.localization.util.TypeProperties">
```

This should be TypeProperties even though its ContentTypeRegistryBean. This service uses some of these classes in content that depend on this class.

**Review by ern — CHANGES_REQUESTED** [review]
I have a PR coming that addresses the underlying issue with the content types not being managed by spring correctly

**Comment by jonespm** [comment]
Closing for #11316

**Pull request closed without merging**